File: backend_ITShop/ITShop.API/ITShop.API/ViewModels/User/UserGetVM.cs
using ITShop.API.Enums;
using ITShop.API.ViewModels.Role;

namespace ITShop.API.ViewModels.User
{
    public class UserGetVM
    {
        public Guid Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string PhoneNumber { get; set; }
        public string UserName { get; set; }
        public string Email { get; set; }
        public Gender Gender { get; set; }
        public List<UserRoleGetVM> Role { get; set; }
    }
}

File: frontend_ITShop/ITShop/src/app/_models/UserGetVM.ts
import {UserRoleGetVM} from "./UserRoleGetVM";

export class UserGetVM {
  public id: string;
  public firstName: string;
  public lastName: string;
  public phoneNumber: string;
  public userName: string;
  public email: string;
  public gender: any;
  public role: UserRoleGetVM[];
  public constructor() {
    this.id  = '';
    this.firstName='';
    this.lastName='';
    this.phoneNumber = '';
    this.userName='';
    this.email='';
    this.role = [];
  }
}

